source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx

```diff
@@ -83,6 +83,7 @@ const SinglePropertyMarker: React.FC<React.PropsWithChildren<SinglePropertyMarke
         fullyAttributedFeature: pointFeature.properties,
       });
     }
+    mapMachine.requestCenterToLocation(latlng);
```

Comment:
why not use the flyto here?

Reply:
'SetView' is better suited to accomplish what is needed in this case sin pan-zoom is not required for this scenario only centering.